xain_fl/grpc/coordinator.py

```diff
@@ -262,30 +275,42 @@ class Coordinator:
     # pylint: disable-msg=dangerous-default-value
     def __init__(
         self,
-        num_rounds: int = 10,
-        required_participants: int = 10,
-        aggregator: Optional[Aggregator] = None,
-        weights: List[ndarray] = [],
-        epochs: int = 0,
+        num_rounds: int = 1,
+        minimum_participants_in_round: int = 1,
+        fraction_of_participants: float = 1.0,
+        weights: List[np.ndarray] = [],
+        epochs: int = 1,
         epoch_base: int = 0,
     ) -> None:
-        self.required_participants: int = required_participants
-        self.participants: Participants = Participants()
-        self.num_rounds: int = num_rounds
-        self.aggregator: Aggregator = aggregator if aggregator else FederatedAveragingAgg()
+        self.minimum_participants_in_round = minimum_participants_in_round
```

Comment:
Can we bring back the typing of the attributes? On the one hand you are right, that the types are inferred from the called functions, input arguments or class instantiating. On the other hand, it is much more understandable if you see the types directly and if you call a function in the `__init__` method, you can immediately recognize typing inconsistencies between attributes and returned types of functions.